ci/pipeline_bm_stemcell.yml

```diff
@@ -0,0 +1,180 @@
+  - task: generate-bm-stemcell-fsa
+    file: bosh-softlayer-tools/ci/tasks/generate-bm-stemcell-fsa.yml
+    params:
+      bm_stemcell_private_key:  {{bm_stemcell_private_key}}
+      bm_root_password: {{bm_root_password}}
+      bm_sjc01: 10.112.249.91
+  - put: stemcell-ubuntu-trusty-bm-fsa
+    params: {from: build/bosh-stemcell-.*-softlayer-baremetal.fsa}
+
+
+resources:
+- name: bosh-src
+  type: git
+  source:
+    uri: https://github.com/jianqiu/bosh.git
+    branch: baremetal_stemcell
+    private_key: {{github_promote_private_key}}
+
+- name: published-stemcell
+  type: bosh-io-stemcell
+  source:
+    name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
+    tarball: false
+
+- name: stemcell-version
+  type: semver
+  source:
+    key: stemcell-version
+    bucket: {{s3_candidate_stemcell_bucket}}
+    access_key_id: {{aws_builder_access_key}}
+    secret_access_key: {{aws_builder_secret_access_key}}
+
+- name: stemcell-ubuntu-trusty-raw
+  type: s3
+  source:
+    regexp: bosh-stemcell-([0-9]+)-softlayer-esxi-ubuntu-trusty-raw.tgz
+    bucket: {{s3_candidate_stemcell_bucket}}
+    access_key_id: {{aws_builder_access_key}}
+    secret_access_key: {{aws_builder_secret_access_key}}
+
+- name: stemcell-ubuntu-trusty-bm-fsa
+  type: s3
+  source:
+    regexp: bosh-stemcell-([0-9]+)-softlayer-baremetal.fsa
+    bucket: {{s3_candidate_stemcell_bucket}}
+    access_key_id: {{aws_builder_access_key}}
+    secret_access_key: {{aws_builder_secret_access_key}}
+
+- name: light-stemcell
+  type: s3
+  source:
+    regexp: light-bosh-stemcell-\d+-softlayer-.*\.tgz
+    bucket: {{s3_candidate_stemcell_bucket}}
+    access_key_id: {{s3-access-key-id}}
+    secret_access_key: {{s3-secret-access-key}}
+
+- name: light-stemcell-final
+  type: s3
+  source:
+    regexp: light-bosh-stemcell-\d+-softlayer-.*\.tgz
+    bucket: {{s3_stemcell_bucket}}
+    access_key_id: {{s3-access-key-id}}
+    secret_access_key: {{s3-secret-access-key}}
+
+- name: bosh-softlayer-tools
+  type: git
+  source:
+    uri: git@github.com:mattcui/bosh-softlayer-tools.git
+    branch: master
+    private_key: {{github_promote_private_key}}
+
+- name: bosh-stemcell-softlayer-image
+  type: s3
+  source:
+    regexp: bosh-stemcell-softlayer.vhd
+    bucket: {{s3_candidate_stemcell_bucket}}
+    access_key_id: {{s3-access-key-id}}
+    secret_access_key: {{s3-secret-access-key}}
+
+- name: bosh-cpi-artifacts
+  type: s3
+  source:
+    regexp: bosh-softlayer-cpi-([0-9.]+)\.tgz
+    bucket: dev-bosh-softlayer-cpi-stemcells
+    access_key_id: {{s3-access-key-id}}
+    secret_access_key: {{s3-secret-access-key}}
+
+- name: stemcell-info
+  type: s3
+  source:
+    bucket: {{s3_candidate_stemcell_bucket}}
+    versioned_file: stemcell-info.json
+    access_key_id: {{s3-access-key-id}}
+    secret_access_key: {{s3-secret-access-key}}
+
+- name: bats
+  type: git
+  source:
+    uri: git@github.com:jianqiu/bosh-acceptance-tests.git
+    branch: 256.x-softlayer
```

Comment:
Why are we still branching on `256.x-softlayer`? I believe `257` is released and should have what we need?

/cc @cppforlife confirm?